client/components/static-section.js

```diff
@@ -2,12 +2,17 @@ import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
 import flatten from 'lodash/flatten';
 import ReviewFields from './review-fields';
+import PDFProtocols from '../pages/sections/granted/pdf-protocols';
 
 const StaticSection = ({ section, project, fields = [], isGranted, subsection = false, ...props }) => {
-  const Component = isGranted
+  let Component = isGranted && !props.isFullApplication
     ? (section.granted && section.granted.review) || section.review || ReviewFields
     : section.review || ReviewFields;
 
+  if (props.pdf && section.name === 'protocols') {
+    Component = PDFProtocols
+  }
```

Comment:
This feels like something that might be better done inside the protocol component, as per https://github.com/UKHomeOffice/asl-projects/blob/master/client/pages/sections/granted/protocols.js#L8-L10

The logic gets a bit spread around otherwise.

Reply:
that is a good shout - I didn't even look at that file. so. many. different. versions. of. protocols.

Reply:
That doesn't actually work, as that file is only used if you're not viewing the "full application", which is the case this is covering. This goes back to the problem with no longer having a read only view, and a granted read only view - now there is another type of view which we include in the pdf 🙄

Reply:
Not related to the above comment, but I think L8-14 wants a bit of a refactor to make the conditions clearer.

I'd probably pull the logic out into a `getComponent` function so it can examine each case and return early as appropriate.